docs/.storybook/webpack.config.js

```diff
@@ -68,10 +68,7 @@ module.exports = ({ config }) => {
     },
     {
       test: /\.yaml$/,
-      use: [
-        { loader: require.resolve('json-loader') },
-        { loader: require.resolve('yaml-loader') },
-      ],
+      use: [{ loader: require.resolve('yaml-loader') }],
```

Comment:
This change is caused by a breaking change update in `yaml-loader` dependency.
With the new [0.7.0 version](https://github.com/eemeli/yaml-loader/releases/tag/v0.7.0), parsed files are returned as Javascript instead of JSON as before. This means we can't use the `json-loader` in the chain for `.yaml` files.